storage/core/src/main/java/com/hortonworks/registries/storage/exception/NotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.hortonworks.registries.storage.exception;
+
+public class NotFoundException extends StorageException {
```

Comment:
nit: There are couple of "[a-zA-Z]*NotFoundException", maybe we can call this "StorableNotFoundException" to distinguish them from this exception :)

Reply:
Nice finding. Will fix.